lib/service.go

```diff
@@ -290,7 +290,11 @@ func (m *MevService) methodNotFound(i *rpc.RequestInfo, w http.ResponseWriter) e
 		log.Println("error in method not found: doing request: ", i.Method, err)
 		return err
 	}
-	defer resp.Body.Close()
+	defer func() {
+		if err := resp.Body.Close(); err != nil {
```

Comment:
This seems functionally equivalent to the original version, since the error is ignored? Perhaps what we should do is log when this error happens. I’ll add it